docs/source/docs/quick-start/common-setups.md

```diff
@@ -0,0 +1,40 @@
+# Common Hardware Setups
+
+## Coprocessors
+
+- Orange Pi 5 4GB
+  - Able to process two object detection streams at once while also processing 1 to 2 apriltag streams at 1280x800 (30fps).
```

Comment:
make it known that this is the only device that supports object detection, unless thats elsewhere